src/twisted/protocols/haproxy/test/test_v1parser.py

```diff
@@ -65,7 +65,7 @@ def test_fullParsingSuccess(self) -> None:
         )
         self.assertIsInstance(info.source, address.IPv4Address)
         assert isinstance(info.source, address.IPv4Address)
-        assert isinstance(info.destination, address.IPv4Address)
+        assert isinstance(info.destination, address.IPv4Address)  # type: ignore[unreachable]
```

Comment:
I don't understand this... but I guess this is a way to fight the tyranny of mypy :)